packages/suite/src/actions/wallet/send/sendFormSolanaActions.ts

```diff
@@ -82,17 +82,43 @@ const calculate = (
 // TODO(vl): For custom fees, accept a value from the user in Lamports. Then get the compute unit price by dividing it by the compute budget. See: https://docs.solana.com/proposals/fee_transaction_priority
 // Reference: https://docs.solana.com/transaction_fees
 export const composeTransaction =
-    (formValues: FormState, formState: ComposeActionContext) => () => {
+    (formValues: FormState, formState: ComposeActionContext) => async () => {
         const { account, network, feeInfo } = formState;
         const composeOutputs = getExternalComposeOutput(formValues, account, network);
-        if (!composeOutputs) return Promise.resolve(undefined); // no valid Output
+        if (!composeOutputs) return; // no valid Output
 
         const { output, decimals } = composeOutputs;
 
-        // FeeLevels are read-only
-        const { levels } = feeInfo;
+        let fetchedFee: string | undefined;
+
+        const estimatedFee = await TrezorConnect.blockchainEstimateFee({
+            coin: account.symbol,
+            request: {
+                specific: {
+                    from: account.descriptor,
+                    to:
+                        formValues.outputs.length > 0
+                            ? formValues.outputs[0].address
+                            : account.descriptor,
```

Comment:
Can you please explain this?
1. Is there a use case when solana tx has more then 1 output? Like sending to multiple recipients? 
2. Does the UI allow such a scenario?
3. The fallback to account descriptor seems strange, it reads as "if there is no recipient, send to self" which it weird

Reply:
outcome discussed offline: comment why this is here